packages/protocol/src/mappings/SavingsContract.ts

```diff
@@ -219,11 +220,10 @@ function updateTotalSavings(
     totalSavings = totalSavingsV1.value
   } else {
     let v2Contract = SavingsContractV2.bind(addr)
-    let totalSavingsV2 = v2Contract.try_totalSupply()
-
-    if (!totalSavingsV2.reverted) {
-      totalSavings = totalSavingsV2.value
-    }
+    let exchangeRate = v2Contract.try_exchangeRate()
+    let totalCredits = v2Contract.try_totalCredits() //this doesnt exist, what do here? :thinking:
+    let totalSavingsV2 = integer.fromNumber(exchangeRate).times(integer.fromNumber(totalCredits))
```

Comment:
`exchangeRate` is not a number it's a result with a `reverted` property and  a `value` property, because it's using the `try_` function (though it shouldn't be using that).